syne_tune/optimizer/schedulers/searchers/bayesopt/gpautograd/kernel/base.py

```diff
@@ -226,7 +223,7 @@ def __init__(
                 )
 
     @property
-    def ARD(self) -> float:
+    def ARD(self) -> bool:
```

Comment:
Typehint was wrong?

Reply:
Yes

Reply:
Yes, I see it was being set to a boolean, resolving